samples/nrf9160/http_application_update/src/main.c

```diff
@@ -6,12 +6,15 @@
 #include <stdint.h>
 #include <zephyr.h>
 #include <flash.h>
+#include <bsd.h>
```

Comment:
switch to use `bsd_init` to remove `bsdlib.h`

Reply:
I think `net/bsdlib.h` was better.
`bsd.h` is the library interface but the application should use the OS glue.